processor/routingprocessor/router.go

```diff
@@ -344,7 +340,7 @@ func (r *router[E]) registerExporters(exporters map[config.ComponentID]component
 	for id, exp := range exporters {
 		exporter, ok := exp.(E)
 		if !ok {
-			return fmt.Errorf("the exporter %q isn't a ... exporter", id.String())
+			return fmt.Errorf("the exporter %q isn't a %T exporter", id.String(), new(E))
```

Comment:
Sweet! What does it yield?

Reply:
It will print smth like `the exporter "otlp" isn't a *component.TracesExporter exporter`